### PR TITLE
[Travel] Results and image

### PR DESCRIPTION
**Pull request by icidasset**
`.force-render` 😂

### CODE REVIEW

File: templates/travel/partials/results/results.snip.html [hunk 1]
```diff
@@ -143,6 +142,7 @@
         </div>
         [[={{ }}=]]
       </template>
+      <span class="force-render">&nbsp;</span>
```
Comment (bentranter) [inline templates/travel/partials/results/results.snip.html]: You can prob change this to a `div` and drop the `nbsp`, the reviewer at Google hates `nbsp`'s with his life for some reason

**Review by bentranter — APPROVED** [review]

File: templates/travel/partials/results/results.snip.html (hunk 1 above)
Comment (bentranter) [reply templates/travel/partials/results/results.snip.html]: But if you can't then it's all good, I'll deal with it during the review if it comes up

Comment (icidasset) [reply templates/travel/partials/results/results.snip.html]: Yeah, I can probably make it a div that's 1px high, will give that a try before I merge this. It doesn't seem to work if it's just a div with nothing in it. Super weird stuff.

Comment (icidasset) [reply templates/travel/partials/results/results.snip.html]: Thanks for making me revisit this, works now as a div 👍

**Pull request merged**